### ISSUE TITLE
Can't locate inc/Module/Install.pm

### ISSUE THREAD

**Issue by eserte** [dsully/perl-crypt-openssl-x509#71]
On perl installations where "." is not anymore in `@INC` (i.e. newer perls) building the distribution fails:
```
Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: /var/tmp/cpansmoker-11577/2018110513/CPAN-Reporter-lib-ZMve /opt/perl-5.28.0/lib/site_perl/5.28.0/x86_64-linux /opt/perl-5.28.0/lib/site_perl/5.28.0 /opt/perl-5.28.0/lib/5.28.0/x86_64-linux /opt/perl-5.28.0/lib/5.28.0) at Makefile.PL line 1.
BEGIN failed--compilation aborted at Makefile.PL line 1.
(/opt/perl-5.28.0/bin/perl Makefile.PL INSTALLDIRS=site exited with 512)
```
Probably can be fixed by prepending a `use lib "."`.

**Comment by jonasbn** [comment]
Hi @eserte 

Thanks - I will investigate

**Comment by eserte** [comment]
A possible patch: http://cpan.cpantesters.org/authors/id/S/SR/SREZIC/patches/Crypt-OpenSSL-X509-1.811-issue71.patch

**Comment by jonasbn** [comment]
Thanks - I will review

**Comment by mikequentel** [comment]
Thanks @eserte the patch worked successfully...able to run `patch < Crypt-OpenSSL-X509-1.811-issue71.patch` and it patched the `Makefile.PL`

Then successfully ran:
```/usr/bin/perl Makefile.PL INSTALLDIRS=vendor 'OPTIMIZE=-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables'```

**Comment by jonasbn** [comment]
I just read up on the issue in [this post](https://www.effectiveperlprogramming.com/2017/01/v5-26-removes-dot-from-inc/) by brian d foy. 

So what the patch effectively does is circumvent the security measure implemented in Perl 5.26 - but I guess we need to do this in order to build.

**Comment by jonasbn** [comment]
Patch included in release 1.8.12, just uploaded to PAUSE/CPAN - thanks @eserte
